etc/kayobe/inventory/group_vars/all/stackhpc-lvm

```diff
@@ -0,0 +1,30 @@
+---
+###############################################################################
+# StackHPC LVM Logical Volume (LV) configuration.
+
+# StackHPC LVM lv_swap LV size.
+stackhpc_lvm_lv_swap_size: 16g
+
+# StackHPC LVM lv_root LV size.
+stackhpc_lvm_lv_root_size: 50g
+
+# StackHPC LVM lv_tmp LV size.
```

Comment:
Was there an issue with glance image conversions using /tmp?  We have some users who make very large images, potentially bigger than this.

Reply:
Wouldn't that be using /tmp inside the glance_api container? Which I think would use the container overlay filesystem by default.